### PR TITLE
Added some comments for ts part

### PR DESCRIPTION
**Pull request by Selawen**

### CODE REVIEW

File: faq-site-client/src/components/posts/Post.vue [hunk 1]
```diff
@@ -76,9 +76,9 @@
         },
         props: {
             postHash: Number,
-            isFullPost: Boolean
+            isFullPost: Boolean // check FullPost
         },
-        mounted() {
+        mounted() { // if there's a post, get post. else get post preview
```
Comment (victorheld) [inline faq-site-client/src/components/posts/Post.vue]: Straight up wrong

File: faq-site-client/src/components/posts/Post.vue [hunk 2]
```diff
@@ -47,20 +47,20 @@
...
-    } from "../../../../faq-site-shared/api-calls";
-    import {IPost, IPostReduced, ITopic} from "../../../../faq-site-shared/models";
-    import {Routes} from "../../views/router/router";
+    } from "../../../../faq-site-shared/api-calls"; // importing different post view modes
+    import {IPost, IPostReduced, ITopic} from "../../../../faq-site-shared/models"; // importing posts
+    import {Routes} from "../../views/router/router"; 
     import dataState from "../../store/data";
 
-    interface IBreadCrumbType {
+    interface IBreadCrumbType { // set interface parameters
```
Comment (victorheld) [inline faq-site-client/src/components/posts/Post.vue]: Too verbose

File: faq-site-client/src/components/posts/Post.vue [hunk 3]
```diff
@@ -47,20 +47,20 @@
...
         PostCallBack,
         PostPreviewCallBack,
         PostPreviewRequest,
         PostRequest
-    } from "../../../../faq-site-shared/api-calls";
-    import {IPost, IPostReduced, ITopic} from "../../../../faq-site-shared/models";
-    import {Routes} from "../../views/router/router";
+    } from "../../../../faq-site-shared/api-calls"; // importing different post view modes
+    import {IPost, IPostReduced, ITopic} from "../../../../faq-site-shared/models"; // importing posts
+    import {Routes} from "../../views/router/router";
```
Comment (victorheld) [inline faq-site-client/src/components/posts/Post.vue]: Why the space

File: faq-site-client/src/components/posts/Post.vue [hunk 4]
```diff
@@ -47,20 +47,20 @@
 </template>
 
 <script lang="ts">
-    import Vue from "vue";
+    import Vue from "vue"; // importing vue
```
Comment (victorheld) [inline faq-site-client/src/components/posts/Post.vue]: I mean really?

File: faq-site-client/src/components/posts/Post.vue [hunk 5]
```diff
@@ -47,20 +47,20 @@
...
     import {
         PostCallBack,
         PostPreviewCallBack,
         PostPreviewRequest,
         PostRequest
-    } from "../../../../faq-site-shared/api-calls";
-    import {IPost, IPostReduced, ITopic} from "../../../../faq-site-shared/models";
-    import {Routes} from "../../views/router/router";
+    } from "../../../../faq-site-shared/api-calls"; // importing different post view modes
+    import {IPost, IPostReduced, ITopic} from "../../../../faq-site-shared/models"; // importing posts
```
Comment (victorheld) [inline faq-site-client/src/components/posts/Post.vue]: Wrong again, those are just **I**nterfaces

**Comment by RobbinBaauw** [comment]
Closing as it seems a troll

**Pull request closed without merging**